src/get-started/java-binding.md

```diff
@@ -449,25 +449,27 @@ For more information on using Guice, see the [project wiki][guice-wiki].
 
 ## Testing
 
-### Schema and Operations with Storage
+You can test Exonum services with the help of the [`exonum-testkit`][testkit-maven]
+library. TestKit allows to test transaction execution in the synchronous environment
+by offering simple network emulation (that is, without consensus algorithm and
+network operation involved).
 
-To test the schema and operations with the storage, Exonum provides a
-database that stores the values in the RAM — [`MemoryDb`][Memorydb].
-Before using it in integration tests, it is necessary to load
-a library with the implementation of native methods:
+For using the library include the dependency in your `pom.xml`:
 
-```java
-public class MySchemaIntegrationTest {
-  static {
-    LibraryLoader.load();
-  }
-
-  // Tests.
-}
+``` xml
+<dependency>
+  <groupId>com.exonum.binding</groupId>
+  <artifactId>exonum-testkit</artifactId>
+  <version>0.7.0</version>
+  <scope>test</scope>
+</dependency>
 ```
 
-The plug-in for running tests should be configured to pass
-`java.library.path` system property to JVM:
+`exonum-testkit` will be already included in projects generated with
```

Comment:
I would create `### Project Configuration` section, and put this sentence as the first, followed by sth like: "Existing projects shall be configured as described below".